stan-dev/cmdstan
# Feature/746 vectorize example model

#### Submisison Checklist


#### Summary:

see issue #746 

#### Intended Effect:

example model reflects best practices, explain choice of prior

#### How to Verify:

```
make build;
make examples/bernoulli/bernoulli
./examples/bernoulli/bernoulli sample data file=examples/bernoulli/bernoulli.data.json
./bin/stansummary output.csv
```

estimate of `theta` should be around 0.2, given data in file `bernoulli.data.json`


#### Side Effects:

N/A

#### Documentation:

N/A

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
